daffodil-test/src/test/scala/org/apache/daffodil/section15/choice_groups/TestChoice.scala

```diff
@@ -123,9 +124,6 @@ class TestChoice {
   @Test def test_direct_dispatch_16(): Unit = { runnerCH.runOneTest("direct_dispatch_16") }
   @Test def test_direct_dispatch_17(): Unit = { runnerCH.runOneTest("direct_dispatch_17") }
 
-  //@Test def test_choice_noBranch() { runnerCH.runOneTest("choice_noBranch") } - Test consumes no data, which causes a TDMLError
-
-
```

Comment:
Why did you delete the choice_noBranch test and replace it with a new one? Was something wrong with choice_noBranch? That name feels more appropriate than optionalChoice00, since this isn't testing optional choices.

Reply:
I didn't see the point of using the external xsd. I can changes the name if you like.

Reply:
I renamed it and move it to the basic model.

Reply:
Gotcha. I suspect this was originally split out beause it's not a valid schema and in some cases that can cause issues. But since it doesn't, seems reasonable to put it along side the other schemas. This change is fine with me.